src/test/java/com/prgrms/springbootbasic/voucher/VoucherTypeTest.java

```diff
@@ -0,0 +1,44 @@
+package com.prgrms.springbootbasic.voucher;
+
+import com.prgrms.springbootbasic.common.exception.InvalidVoucherTypeException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class VoucherTypeTest {
+
+    @ParameterizedTest
+    @CsvSource({"fixedAmount,FIXED_AMOUNT", "percent,PERCENT"})
+    @DisplayName("사용자의 Voucher 타입 입력에 따라 Voucher Type을 정할 수 있다")
+    void findCommandByInputValue(String voucherTypeInput, String expectedVoucherType) {
```

Comment:
```suggestion
    void findCommandByInputValue(String voucherTypeInput, VoucherType expectedVoucherType) {
```
이렇게해도 파라미터가 매핑될거에요 ㅎㅎ

Reply:
오우 진짜네요! 😄 내부적으로 파라미터로 넘긴 enum class의 valueOf를 호출하나...? 신기하군요 감사합니다!!